app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -40,24 +37,42 @@ import FileBase64CacheContext from '../context/file-base64-cache';
  *
  * @type {number}
  */
-const DEFAULT_ACCEPTABLE_GLARE_SCORE = 50;
+const ACCEPTABLE_GLARE_SCORE = 50;
 
 /**
  * The minimum sharpness score value to be considered acceptable.
  *
  * @type {number}
  */
-const DEFAULT_ACCEPTABLE_SHARPNESS_SCORE = 50;
+const ACCEPTABLE_SHARPNESS_SCORE = 50;
 
 /**
  * The minimum file size (bytes) for an image to be considered acceptable.
  *
  * @type {number}
  */
-const DEFAULT_ACCEPTABLE_FILE_SIZE_BYTES =
-  process.env.ACUANT_MINIMUM_FILE_SIZE === undefined
-    ? 250 * 1024
-    : Number(process.env.ACUANT_MINIMUM_FILE_SIZE);
+export const ACCEPTABLE_FILE_SIZE_BYTES = 250 * 1024;
+
+/**
+ * Returns attribute value to assign to the input field, depending on the current environment.
+ *
+ * @return {string[]=} Minimum file size, in bytes.
+ */
+export function getInputAccept() {
+  return process.env.NODE_ENV === 'production' ? ['image/*'] : undefined;
```

Comment:
This will also unfortunately not work as expected in dev and int. There the `NODE_ENV` is `production`.

Reply:
> This will also unfortunately not work as expected in dev and int. There the `NODE_ENV` is `production`.

Yeah, I suppose not 😞  If we're okay to make `doc_auth_vendor` ([and possibly `acuant_simulator`](https://github.com/18F/identity-idp/blob/19ebc9c966a8d8452e52c446ddb7f92261e9850b/app/services/doc_auth/client.rb#L23)) available to the browser, I think that would be the more reliable check?

Reply:
Yeah that seems fine to expose as long as we continue to enforce on the server too

Reply:
Updated in 45091d2, using a similar pattern to what we'd done previously to "pass in arguments" through the root element. This means we're not providing the actual environment variables, and can also consider the additional logic of `DocAuth::Client.doc_auth_vendor`.